pkg/sql/schema_changer_test.go

```diff
@@ -476,6 +476,48 @@ func TestRollbackOfAddingTable(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestUniqueViolationsAreCaught(t *testing.T) {
```

Comment:
This test is a bit unrelated, but I've had it sitting around uncommitted and it feels useful to have.